tests/foreman/api/test_contentview.py

```diff
@@ -187,14 +181,13 @@ def test_cv_associate_composite_dupe_repos_negative(self):
         self.assertEqual(len(content_view.read().repository), 0)
 
     @run_only_on('sat')
-    def test_cv_associate_composite_dupe_modules_negative(self):
+    def test_negative_associate_composite_dupe_modules(self):
```

Comment:
this does not look like a composite test? may be remove composite from the name?